inc/az_ulib_ipc_api.h

```diff
@@ -214,6 +219,35 @@ AZ_NODISCARD az_result az_ulib_ipc_try_get_interface(
     az_ulib_version_match_criteria match_criteria,
     az_ulib_ipc_interface_handle* interface_handle);
 
+/**
+ * @brief   Try get a capability index in the interface by the name from the IPC.
+ *
+ * This API tries to find an capability that fits the provided name in the interface. It returns
+ * the capability index.
+ *
+ * @param[in]   interface_handle    The #az_ulib_ipc_interface_handle with the interface handle.
+ *                                  It cannot be `NULL`. Call az_ulib_ipc_try_get_interface() to
+ *                                  get the interface handle.
+ * @param[in]   name                The `az_span` with the interface name.
+ * @param[out]  capability_index    The #az_ulib_capability_index* with the memory to store
```

Comment:
this was already present before this PR, correct? not entirely sure what it indicates.